recipes/miktex-portable/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+
+7za x miktex-portable-%PKG_VERSION%.exe -o%LIBRARY_PREFIX%\miktex-portable
+if errorlevel 1 exit 1
+
+
+rem SCRIPTS dir should already be created by 7za install
+mkdir "%SCRIPTS%"
+if errorlevel 1 exit 1
+
+rem latex tools must be run from miktex tree
+for %%f in ("%LIBRARY_PREFIX%\miktex-portable\miktex\bin\*.exe") do (
+	echo @%%~dp0\..\Library\miktex-portable\miktex\bin\%%~nf %%* >> "%SCRIPTS%\%%~nf.bat"
```

Comment:
You mean `echo @%%~dp0\..\Library\...`? This is called at runtime, so `%library_prefix%` is no set.